packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -47,7 +49,12 @@ import '../../tools/types.js';
 const start = async (zcf, privateArgs) => {
   const { timer, POLL_INTERVAL, brandIn, brandOut, unitAmountIn } =
     zcf.getTerms();
+  assert(brandIn, 'missing brandIn');
+  assert(brandOut, 'missing brandOut');
+  assert(POLL_INTERVAL, 'missing POLL_INTERVAL');
+  assert(timer, 'missing timer');
```

Comment:
My eyes are starting to expect shape checks for this sort of thing. It would let us check that brands are remotables, for example.

not critical, but I have to keep in mind that `brandIn` might be some scary JavaScript proxy monster thing that goes bump in the night as I review this...